simulate.py

```diff
@@ -61,6 +61,8 @@ def get_contigs(self):
     parser.add_argument("--max_pass", help="Maximum passes", type=int, default=1000000000)
     parser.add_argument("--output_polymerase_read", help="shall we simulate polymerase read (including adapter)?", action='store_true')
     parser.set_defaults(output_polymerase_read=False)
+    parser.add_argument("--estimate_np_by_length", help="shall we estimate number of passes by sampled length?", action='store_true')
+    parser.set_defaults(estimate_np_by_length=False)
```

Comment:
Do not need this. `action = 'store_true'` implies that it is `False` by default.